README.md

```diff
@@ -51,7 +51,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link] (Not deployed)
+- [Live Demo Link](https://metaverse-nabeel.github.io/Portfolio-Project/)
```

Comment:
In my previous PR this was the optional change required that's why added link to deployed app here.